recipes/essentials/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.25)
+
+project(test_package CXX) # if the project uses c++
+
+find_package(essentials REQUIRED CONFIG)
+
+include_directories(${essentials_INCLUDE_DIRS})
```

Comment:
```suggestion
```

Reply:
Removed the _include_directories_ command.